inst/shiny/server.R

```diff
@@ -63,83 +68,83 @@ server <- shinyServer(function(input, output, session) {
   #For theming
   #bs_themer()
     sessionid <- random_id(n = 1)
```

Comment:
Is this the only use of the ids package? If so, we might rather use something else to keep the total number of dependencies/suggests low.

Reply:
That is the only use of the ids package. I am not positive that hashing the sessionInfo will work on the online shiny app because I don't know if it actually generates a new sessionInfo at every start. Looking at sessionInfo on my computer:
![Uploading image.png…]()

I suspect that this would not be unique for every time a user logs on (which is what we want). I don't know that it would even update on a desktop every time a user opens the app either. We might want to use something else like the ids package as the default for that input.

Reply:
Yes of course, the session info won't work for shinyapps.io. This is why I made it an argument of the `share_spec()` function. With this, we can just feed whatever ID we want. But what about just taking a random number using R's basic tools. We don't need something cryptogaphically safe here.